sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -580,6 +580,11 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val BUCKETING_MAX_BUCKETS = buildConf("spark.sql.bucketing.maxBuckets")
```

Comment:
Make it consistent with `spark.sql.sources.bucketing.enabled`? rename it to `spark.sql.sources.bucketing.maxBuckets`?

Reply:
Oh... did it change or I overlooked 'sources'? Sure I will change!